[2025-07-11T07:55:06.524+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-11T07:55:06.552+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: deepai_crawler_10min.crawl_hardware scheduled__2025-07-11T07:40:00+00:00 [queued]>
[2025-07-11T07:55:06.570+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: deepai_crawler_10min.crawl_hardware scheduled__2025-07-11T07:40:00+00:00 [queued]>
[2025-07-11T07:55:06.572+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-07-11T07:55:06.638+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): crawl_hardware> on 2025-07-11 07:40:00+00:00
[2025-07-11T07:55:06.656+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=135) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-11T07:55:06.660+0000] {standard_task_runner.py:63} INFO - Started process 150 to run task
[2025-07-11T07:55:06.658+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'deepai_crawler_10min', 'crawl_hardware', 'scheduled__2025-07-11T07:40:00+00:00', '--job-id', '1544', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmpe1a3xpsm']
[2025-07-11T07:55:06.669+0000] {standard_task_runner.py:91} INFO - Job 1544: Subtask crawl_hardware
[2025-07-11T07:55:06.765+0000] {task_command.py:426} INFO - Running <TaskInstance: deepai_crawler_10min.crawl_hardware scheduled__2025-07-11T07:40:00+00:00 [running]> on host 13e6c13b5a13
[2025-07-11T07:55:06.937+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='deepai_crawler_10min' AIRFLOW_CTX_TASK_ID='crawl_hardware' AIRFLOW_CTX_EXECUTION_DATE='2025-07-11T07:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-11T07:40:00+00:00'
[2025-07-11T07:55:06.940+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-11T07:55:06.970+0000] {dags.py:178} INFO - B·∫Øt ƒë·∫ßu crawl category: hardware
[2025-07-11T07:55:06.980+0000] {media_manager.py:52} DEBUG - Queue: Media upload queue is empty, waiting for new jobs
[2025-07-11T07:55:07.418+0000] {crawler_deeplai.py:51} INFO - ƒê√£ k·∫øt n·ªëi th√†nh c√¥ng v·ªõi MongoDB: deeplearning_ai_news
[2025-07-11T07:55:07.502+0000] {client.py:1980} DEBUG - Getting prompt '3 Points-label:production'
[2025-07-11T07:55:07.503+0000] {client.py:1984} DEBUG - Prompt '3 Points-label:production' not found in cache or caching disabled.
[2025-07-11T07:55:07.504+0000] {client.py:2068} DEBUG - Fetching prompt '3 Points-label:production' from server...
[2025-07-11T07:55:07.982+0000] {media_manager.py:52} DEBUG - Queue: Media upload queue is empty, waiting for new jobs
[2025-07-11T07:55:08.983+0000] {media_manager.py:52} DEBUG - Queue: Media upload queue is empty, waiting for new jobs
[2025-07-11T07:55:09.699+0000] {log.py:232} WARNING - 2025-07-11 07:55:09,699 - httpx - INFO - HTTP Request: GET http://host.docker.internal:4000/api/public/v2/prompts/3%20Points?label=production "HTTP/1.1 200 OK"
[2025-07-11T07:55:09.699+0000] {_client.py:1025} INFO - HTTP Request: GET http://host.docker.internal:4000/api/public/v2/prompts/3%20Points?label=production "HTTP/1.1 200 OK"
[2025-07-11T07:55:09.710+0000] {crawler_deeplai.py:465} INFO - üöÄ ƒêang b·∫Øt ƒë·∫ßu qu√° tr√¨nh crawl v·ªõi Load More...
[2025-07-11T07:55:09.984+0000] {media_manager.py:52} DEBUG - Queue: Media upload queue is empty, waiting for new jobs
[2025-07-11T07:55:10.984+0000] {media_manager.py:52} DEBUG - Queue: Media upload queue is empty, waiting for new jobs
[2025-07-12T20:57:48.914+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-12T20:57:48.931+0000] {taskinstance.py:2066} INFO - Dependencies not met for <TaskInstance: deepai_crawler_10min.crawl_hardware scheduled__2025-07-11T07:40:00+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2025-07-12T20:57:48.944+0000] {taskinstance.py:2066} INFO - Dependencies not met for <TaskInstance: deepai_crawler_10min.crawl_hardware scheduled__2025-07-11T07:40:00+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2025-07-12T20:57:48.947+0000] {local_task_job_runner.py:163} INFO - Task is not able to be run
